src/layouts/Form/FieldWrapper.js

```diff
@@ -68,6 +69,10 @@ class FieldWrapper extends React.Component {
     error: null,
   };
 
+  componentDidMount() {
+    warning(true, 'DEPRECATED: the Field system is now deprecated. Please use Flow instead. (from FieldWrapper)');
```

Comment:
That's a nice idea. I think this is probably good enough for this round, but next time things get deprecated I'll definitely look into it.